solr_http.go

```diff
@@ -307,8 +308,8 @@ func defaultReadClient(cert string, https bool) (HTTPer, error) {
 	return cli, nil
 }
 
-func getTLSConfig(certPath string) (*tls.Config, error) {
-	tlsConf := &tls.Config{InsecureSkipVerify: true}
+func getTLSConfig(certPath string, verify bool) (*tls.Config, error) {
+	tlsConf := &tls.Config{InsecureSkipVerify: verify}
```

Comment:
this is confusing, when InsecureSkipVerify is true, verification isn't happening.  i'd expect verify = true to set InsecureSkipVerify = false